GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -455,82 +435,12 @@ private void SetRowCountAndSelectRowsIfReady(int rowCount)
             SelectRowsIfReady(rowCount);
         }
 
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity", Justification = "It looks like such lock was made intentionally but it is better to rewrite this")]
-        private async Task RunBackgroundAsync(CancellationToken cancellationToken)
+        private void UpdateVisibleRowRange()
         {
-            if (LicenseManager.UsageMode == LicenseUsageMode.Designtime)
-            {
-                // Don't run background operations in the designer.
-                return;
-            }
```

Comment:
Has this been removed? Have you verified this control still renders in the designer correctly?

Reply:
If I'm not mistaken the control can't be opened in the designer already (current `master` branch).